sensorlib/doc/CarlaCDASimAPI.md

```diff
@@ -0,0 +1,208 @@
+# Carla CDA Sim API
```

Comment:
Same comment about the naming here, we can revert this to Sensor Library API or what ever you had initially called this since it is not intrinsically connect to CDA Sim at all.